### ISSUE TITLE
Accessing OPDS sources on amd64 fails

### ISSUE THREAD

**Issue by everdred** [koreader/koreader#7499]
* KOReader version 2021.03
* Device: amd64

#### Issue
When I try to access any OPDS source, koreader asks if I want to turn on wifi. I am connected to the internet, not to mention all other network functionality in koreader works, including pushing and pulling progress, accessing a Wallabag server, etc.  Regardless of the source I choose, or whether I select "Cancel" or "Turn on" at the prompt, I cannot access the OPDS source.

This doesn't reproduce for me on the latest nightly on Kobo or Android.

**Comment by Frenzie** [comment]
By amd64, do you mean the Debian package or the AppImage?

I can't reproduce that regardless, nor is there a reason it should behave any different than Wallabag etc.

**Comment by everdred** [comment]
I originally noticed this while using the Debian package, but I just reproduced it with the AppImage as well. I happened to launch the AppImage from the terminal and noticed this line coincide with the error:

`sh: 1: /sbin/route: not found`

This is on a relatively fresh Debian Unstable installation.

**Comment by Frenzie** [comment]
That error would make sense (apparently `net-tools` isn't part of the base packages), but not that you only get it on OPDS.

**Comment by everdred** [comment]
I don't normally use it, but I see now it also reproduces when I try connecting to a Calibre's "wireless device connection" functionality using the Calibre plugin. The server for each is running on 127.0.0.1, though the same error also occurs when I try accessing the OPDS of Feedbooks or Gutenberg.

Not sure if it's helpful or related, but when I go to the cog menu > Network  > Network Info I get a blank SSID, Gateway ping successful, but `sh: 1: ifconfig: not found` in the terminal.

**Comment by pazos** [comment]
yup. `ifconfig` and `route` are both part of `net-tools` and that's being deprecated in all major distros in favor of `iproute2`, which has the command `ip` that can do the same.

Arch was one of the first distros deprecating it in 2011: https://archlinux.org/news/deprecation-of-net-tools/. Probably it's time now for debian.

On the debian package we can include `net-tools` as a dependency. Not sure how to handle the appimage thingy.

**Comment by Frenzie** [comment]
It's probably worthwhile to be able to deal with both instead for the
future.

The part I don't understand (haven't investigated) is that I don't have
net-tools on my fresh Debian testing either, but stuff just loads?
